OPHD/UI/PopulationPanel.cpp

```diff
@@ -66,6 +66,8 @@ PopulationPanel::PopulationPanel(Population* pop, PopulationPool* popPool) :
 		imageCache.load("ui/skin/window_bottom_right.png")
 	}
 {
+	mPopulation = pop;
+	mPopulationPool = popPool;
```

Comment:
These two lines should probably have been part of the previous commit. It might also have been helpful to use the constructor init-list, rather than the constructor body. That would be important for the switch to reference syntax, since the constructor body is too late to initialize references (or `const` values).

Note that a pointer to const data is not itself const. The pointer can still be modified, even if the data is const. A reference is like a const pointer, in that the pointer itself can not be modified after initialization.